lib/blackbeard/feature.rb

```diff
@@ -4,12 +4,22 @@ class Feature < Storable
     include Chartable
 
     set_master_key :features
-    string_attributes :name, :description, :status
+    string_attributes :name, :description, :status, :threshold_array_serialized
```

Comment:
I like this approach, but I would make it a son attribute. Also could use a better name. :threshold_moduli ?